system-parachains/encointer/src/lib.rs

```diff
@@ -585,12 +597,65 @@ impl pallet_encointer_democracy::Config for Runtime {
 
 parameter_types! {
 	pub const TreasuriesPalletId: PalletId = PalletId(*b"trsrysId");
+
+	pub const AnyNetwork: Option<NetworkId> = None;
 }
+
+pub type TransferOverXcm = crate::treasuries_xcm_payout::TransferOverXcm<
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	AccountId,
+	// Needs to match AssetKind of the encointer_treasuries::Config
+	VersionedLocatableAsset,
```

Comment:
Instead of adding this comment, you could create a common type alias that's used in both places